### ISSUE TITLE
batch size selection

### ISSUE THREAD

**Issue by XuYunqiu** [ppengtang/pcl.pytorch#22]
Hi, thanks for sharing your implementation. I have a question about the batch size. Why batch size of 2 is selected? Batch size of 2 is also chosen by OICR. Have your ever tired batch size of 1, or other batch size larger than 2?

**Comment by ppengtang** [comment]
For the Caffe implementation on the PASCAL VOC datasets, we haven't tried other batch sizes and just follow Fast R-CNN to set batch size to 2. For this PyTorch implementation, we set batch size to 4 because we found that batch size 4 obtains higher accuracy than other choices.

**Comment by XuYunqiu** [comment]
> For the Caffe implementation on the PASCAL VOC datasets, we haven't tried other batch sizes and just follow Fast R-CNN to set batch size to 2. For this PyTorch implementation, we set batch size to 4 because we found that batch size 4 obtains higher accuracy than other choices.

Thanks for sharing your experiments with me. I write a Pytorch implementation with batch size 1. A disappoint result is obtained with ROI Align. However, it works well with ROI Pooling. Probably, this is caused by different hyper-parameters (such as  batch size), or some other randomness.

**Comment by ppengtang** [comment]
No problem. Actually, many people have told me that ROI Align doesn't work for weakly supervised object detection.

**Comment by XuYunqiu** [comment]
> No problem. Actually, many people have told me that ROI Align doesn't work for weakly supervised object detection.

Okay, thanks a lot anyway.
